docs/src/docs/asciidoc/user/serializers-copiers.adoc

```diff
@@ -15,7 +15,7 @@ endif::notBuildingForSite[]
 
 While Ehcache is a Java cache, it cannot always store its mapping as Java objects.
 
-The on-heap store is capable of storing cached objects either by reference (where the given key and value references
+The <<caching-concepts.adoc#storage-tiers,on-heap store>> is capable of storing cached objects either by reference (where the given key and value references
```

Comment:
That does not work, as the end result are html files. See other links between sections across pages for the proper syntax.